bricksrc/quantities.py

```diff
@@ -1,109 +1,565 @@
-        "subclasses": {
+        "subconcepts": {
             "Electric_Current": {
-                "subclasses": {
-                    "Current_Angle": {},
-                    "Current_Magnitude": {},
-                    "Current_Imbalance": {},
-                    "Current_Total_Harmonic_Distortion": {},
-                    "Alternating_Current_Frequency": {},
+                OWL.sameAs: QUDTQK["ElectricCurrent"],
+                "subconcepts": {
+                    "Current_Angle": {
+                        SKOS.definition: Literal(
+                            "The angle of the phasor representing a measurement of electric current"
+                        ),
+                        QUDT.applicableUnit: [
+                            UNIT.ARCMIN,
+                            UNIT.ARCSEC,
+                            UNIT.DEG,
+                            UNIT.GON,
+                            UNIT.GRAD,
+                            UNIT.MIL,
+                            UNIT.RAD,
+                            UNIT.MicroRAD,
+                            UNIT.MilliRAD,
+                            UNIT.MilliARCSEC,
+                            UNIT.REV,
+                        ],
+                        QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                        SKOS.definition: Literal("Angle of current phasor"),
+                        RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                        RDFS.label: Literal("CurrentAngle"),
+                        SKOS.broader: BRICK.PhasorAngle,
+                    },
+                    "Current_Magnitude": {
+                        SKOS.definition: Literal(
+                            "The magnitude of the phasor representing a measurement of electric current"
+                        ),
+                        QUDT.applicableUnit: [
+                            UNIT.A,
+                            UNIT.MilliA,
+                            UNIT.MicroA,
+                            UNIT.KiloA,
+                        ],
+                        SKOS.definition: Literal("Magnitude of current phasor"),
+                        RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                        RDFS.label: Literal("CurrentMagnitude"),
+                        SKOS.broader: BRICK.PhasorMagnitude,
+                    },
+                    "Current_Imbalance": {
+                        SKOS.definition: Literal(
+                            "The percent deviation from average current"
+                        ),
+                        QUDT.applicableUnit: [UNIT.PERCENT],
+                        QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                        RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                        RDFS.label: Literal("CurrentImbalance"),
+                        SKOS.broader: BRICK.Dimensionless,
+                    },
+                    "Current_Total_Harmonic_Distortion": {
```

Comment:
This is not a subconcept of `Current` but rather `Total_Harmonic_Distortion`, if exists.

Reply:
Are you sure? Total Harmonic Distortion is a quantity of electrical signals, e.g. https://en.wikipedia.org/wiki/Total_harmonic_distortion

Reply:
You are right in that it's a quality of electrical signals, but not a type of electric current (i.e., flow rate).

Reply:
Flow rate has nothing to do with electric current, so I'm not sure where that confusion is coming from. What's the suggested edit here?